bin/varnishlog/varnishlog_options.h

```diff
@@ -56,11 +56,12 @@
 	VOPT("w:", "[-w <filename>]", "Output filename",		\
 	    "Redirect output to file. The file will be overwritten"	\
 	    " unless the -a option was specified. If the application"	\
-	    " receives a SIGHUP in daemon mode the file will be "	\
-	    " reopened allowing the old one to be rotated away. The"	\
-	    " file can then be read by varnishlog and other tools with"	\
-	    " the -r option, unless the -A option was specified. This"	\
-	    " option is required when running in daemon mode. If the"	\
+	    " application runs as a daemon (-D option) or adds a"	\
```

Comment:
"application application" spotted.

Reply:
Oh okay it's for the merge with OSS VC.
Shall I do anything here to help you ?
( comment apply to the next remark as well ! )

Reply:
I'm pretty sure I'm the one who borked this one.

Reply:
I'm sorry to confirm that - I compared your branch, mine and VC master just to be sure I did not do anything wrong.
Thanks for your work !